google-cloud-pubsub/google-cloud-pubsub.gemspec

```diff
@@ -23,6 +23,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency "google-cloud-pubsub-v1", "~> 0.0"
 
   gem.add_development_dependency "autotest-suffix", "~> 1.1"
+  gem.add_development_dependency "avro", "~> 1.10"
```

Comment:
Is this introducing the dependency because of the tests and/or samples? I'm a little nervous introducing the dependency for the entire library just because of those. For example, one who is using proto isn't going to want to have a dependency on the avro library.

Reply:
It's just for tests in this case, and won't impact the published package.

> Development dependencies aren't installed by default and aren't activated when a gem is required.

https://guides.rubygems.org/specification-reference/#add_development_dependency